LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -144,24 +144,15 @@ class LockboxXCUITests: BaseTestCase {
         waitforExistence(app.buttons["finish.button"], timeout: 10)
         app.buttons["finish.button"].tap()
         waitforExistence(app.navigationBars["firefoxLockbox.navigationBar"])
-    }
-
-    func test5SendUsageDataSwitch() {
-//        navigator.goto(Screen.SettingsMenu)
-//        // Disable the send usage data
-//        navigator.performAction(Action.SendUsageData)
-//        XCTAssertEqual(app.switches["sendUsageData.switch"].value as? String, "0")
-//
-//        // Enable it again
-//        navigator.performAction(Action.SendUsageData)
-//        XCTAssertEqual(app.switches["sendUsageData.switch"].value as? String, "1")
+        sleep(5)
```

Comment:
In theory this is only needed to run tests locally, on BB tests are working without adding this...let's try to have them working without this...

Reply:
Oops, I misunderstood this. Removing.